pkg/logutil/node.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// Package logutil implements various log utilities.
+package logutil
+
+import (
+	"fmt"
+	"strings"
+)
+
+// DefaultNodeLogLevel is the default node log level.
+var DefaultNodeLogLevel = "info"
+
+func AvalanchegoToCorethLogLevel(logLevel string) (string, error) {
```

Comment:
Why do we need this?

Reply:
The server received an avalanchego log level but shall give a go-ethereum log level to coreth. Without also controlling coreth logs, there is no complete control of log level. Not considering other plugins for the moment.

Reply:
I don't think this should be here. If it's needed, it should go into avalanchego, otherwise every library using it has to do it.
Also, I have never seen that we config used for avalanchego is applied automagically to the cchain...That's why different configs exists.

I don't think  we should do this....